pkg/build/templates/extract-iso.sh.tpl

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -euo pipefail
+
+#  Template Fields
+#  IsoExtractDir - Full path to the directory (under the build directory) where the ISO should be extracted
+#  RawExtractDir - Full path to the directory (under the build directory) where the RAW image should be
+#                  unsquashed
+#  IsoSource - Full path to the ISO to extract
+
+ISO_EXTRACT_DIR={{.IsoExtractDir}}
+RAW_EXTRACT_DIR={{.RawExtractDir}}
+ISO_SOURCE={{.IsoSource}}
```

Comment:
Is there a benefit in using bash variables instead of the templated ones? Seems like a repetition to me which is making the script unnecessarily longer.

Reply:
I went back and forth on this. Doing it this way helps separate the tests verifying the template is correctly filled out from tests that the script itself works. The division here is that testing this function only cares that the correct values are passed into the template, not that the script itself is correct.